extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -126,6 +126,10 @@ export class FileConfigPage extends ImportPage {
 		return true;
 	}
 
+	public setupNavigationValidator(): void {
```

Comment:
Instead of this I'd suggest just putting this in the base class and if superclasses want to override it they can choose to do so. (and maybe check whether the other navigation validators are actually necessary while you're looking at it too, maybe you can remove this completely)